sbt-bundle-tester/src/main/configuration/frontend/bundle.conf

```diff
@@ -0,0 +1,21 @@
+version    = "1.0.0"
+name       = "override"
+system     = "override-1.0.0"
+nrOfCpus   = 1.0
+memory     = 67108864
+diskSpace  = 5000000
+roles      = ["frontend"]
+components = {
+  "override-1.0.0" = {
+    description      = "override"
+    file-system-type = "universal"
+    start-command    = ["override-1.0.0/bin/override", "-J-Xms67108864", "-J-Xmx67108864"]
+    endpoints        = {
+      "override" = {
+        bind-protocol  = "http"
+        bind-port = 0
+        services  = ["http:/override"]
+      }
+    }
+  }
+}
```

Comment:
Same deal as for backend i.e. a smaller example illustrating that you don't have to type out all of the config.